scheduler/src/main.rs

```diff
@@ -4,7 +4,7 @@ use nettu_scheduler_api::Application;
 use nettu_scheduler_infra::setup_context;
 use telemetry::init_subscriber;
 
-#[actix_web::main]
+#[tokio::main]
```

Comment:
Needed for the OTLP exporter (batch)

For the app/Actix_web, it doesn't change much ([here](https://github.com/actix/actix-web/blob/master/actix-web/MIGRATION-4.0.md#actix_webmain-and-tokiomain))